protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/copy_data_types.rs

```diff
@@ -87,6 +103,11 @@ impl Fixed for u64 {
     const SIZE: usize = 8;
 }
 
+/// Macro to implement the `Sv2DataType` trait for unsigned integer types.
+///
+/// Simplifies encoding and decoding for various unsigned integer types, making them
+/// compatible with the SV2 protocol. Each implementation uses the little-endian byte order for
+/// serialization and deserialization, ensuring consistency across platforms.
```

Comment:
I cannot find this anywhere when I render Rust Docs via `cargo doc`

no point in using `///` here